lib/Minz/Log.php

```diff
@@ -55,7 +55,8 @@ public static function record ($information, $level, $file_name = null) {
 				$level_label = 'debug';
 				break;
 			default :
-				$level_label = 'unknown';
+				$level = LOG_INFO;
```

Comment:
Not something we actually use, but just for code robustness, as we use `$level` a few lines below, and it is important that it is a valid value.